prow/config/postsubmits.yaml

```diff
@@ -417,6 +417,53 @@ postsubmits:
             cpu: "2"
             memory: 1Gi
 
+  maistra/istio-operator:
+  - name: update-charts-in-2.1
+    decorate: true
+    path_alias: maistra.io/istio-operator
+    skip_report: false
+    branches:
+    - ^maistra-1.0$
+    - ^maistra-1.1$
+    - ^maistra-2.0$
+    # Allow for testing
+    - playground
+    labels:
+      preset-github: "true"
+    extra_refs:
+    - base_ref: main
+      org: maistra
+      path_alias: maistra.io/test-infra
+      repo: test-infra
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.1"
+        imagePullPolicy: Always
+        command:
+        - ../test-infra/tools/automator.sh
+        - -o maistra
+        - -r istio-operator
+        - -f /creds-github/github-token
+        - -c make clean update-1.0-charts update-1.1-charts update-2.0-charts
```

Comment:
This is not going to work as intended. Let's say 1.1 branch was updated. Then this job will run on the 1.1 branch, and these targets do not exist. You will want to run this job on the 2.1 branch, right?

Reply:
You probably want to add `-b maistra-2.1` to this script.

Reply:
Also, add `-l auto-merge`.
And see this comment: https://github.com/maistra/istio-operator/pull/746#issuecomment-862489962

This is part of MAISTRA-2445.

Reply:
Thanks. I missed the fact that the target branch isn't defined anywhere.